src/_guides/language/effective-dart/design.md

```diff
@@ -476,6 +476,25 @@ can delete dead code. It can't do that if the member is public because it
 doesn't know if any code outside of its view is using it.
 
 
+### CONSIDER declaring multiple classes in the same library.
+
+Java and C# tie the organization of files to the organization of classes — each
```

Comment:
> ... classes — each ...

I usually prefer using `&mdash;` since it is then clearer that it _is_ an mdash.

Reply:
Done. Fixed the others too (in this file). Also consistently fixed them to not use spaces on either side. (I personally prefer thin spaces around em dashes because no spaces looks too tight to me, but maybe I'm a little too obsessed with this kind of minutia...)